tests/integration/browser.worker.js

```diff
@@ -8,7 +8,17 @@ if (!sourceFile) {
   sourceFile = '../../dist/' + sourceFile[1];
 }
 
-if (typeof window.Worker === 'function' && window.chrome) {
+// only running in Chrome and Firefox due to various bugs.
+// IE: https://connect.microsoft.com/IE/feedback/details/866495
+// Safari: doesn't have IndexedDB or WebSQL in a WW
+// NodeWebkit: not sure what the issue is
+
+var isNodeWebkit = typeof window !== 'undefined' &&
+  typeof process !== 'undefined';
+
+if (typeof window.Worker === 'function' &&
+    !isNodeWebkit &&
+    (window.chrome || /Firefox/.test(navigator.userAgent))) {
```

Comment:
I simplified our `browser.worker.js` browser support checks. New rule: the tests are only run in Chrome and Firefox.